packages/contracts-bedrock/scripts/ChainAssertions.sol

```diff
@@ -146,4 +147,11 @@ library ChainAssertions {
         require(ProtocolVersion.unwrap(versions.required()) == cfg.requiredProtocolVersion());
         require(ProtocolVersion.unwrap(versions.recommended()) == cfg.recommendedProtocolVersion());
     }
+
+    /// @notice Asserts that the SuperchainConfig is setup correctly
+    function checkSuperchainConfig(Types.ContractSet memory _proxies, DeployConfig _cfg) internal view {
+        SuperchainConfig superchainConfig = SuperchainConfig(_proxies.SuperchainConfig);
+        require(superchainConfig.guardian() == _cfg.portalGuardian());
```

Comment:
`_cfg.portalGuardian()` is currently used in both the superchainConfig and the portal. 
I will rename this when it is removed from the portal.